be/src/column/binary_column.h

```diff
@@ -272,6 +272,10 @@ class BinaryColumn final : public ColumnFactory<Column, BinaryColumn> {
         return ss.str();
     }
 
+    bool exceed_capacity_limit() const override {
+        return _bytes.size() >= UINT32_MAX || _offsets.size() >= UINT32_MAX || _slices.size() >= UINT32_MAX;
```

Comment:
the order of things's size, which use for compare with UINT32_MAX is mismatch with the same business logic in be/src/column/object_column.h, specific change is https://github.com/StarRocks/starrocks/pull/95/files#diff-d812e4f6fbc0b6de2dda9a257444c4d30684d5612ac035250899b23ebfdb4441R194
Is there any consideration to do so? May be we can put the most likely true expression in the left to take full advantage of short-circuit calculation.

Reply:
bytes are most likely to exceed the limit. This place is not a high-frequency call, a Join will only be executed once

Reply:
Just a defense, false in most cases